charter.html

```diff
@@ -377,6 +377,13 @@ <h2 id=process>Process</h2>
 <a href=https://www.w3.org/community/about/agreements/cla/>W3C Community
 Contributor License Agreement (CLA)</a>.
 
+<p>The Community Group will not work on or publish a <a
```

Comment:
I'd rather see this just say "publish", not "work on or publish".

Reply:
Works for me, done.